src/components/Switcher/ThemeSwitcher.module.scss

```diff
@@ -0,0 +1,16 @@
+.switcher {
```

Comment:
Please, center the buttons vertically
<img width="106" alt="Screenshot 2023-10-16 at 20 17 47" src="https://github.com/tkachenya41/Frontend-project/assets/47400885/3d788d5c-394f-4233-8fa0-c518fa0ca350">

Reply:
Please add cursor: pointer;